configure.ac

```diff
@@ -355,6 +355,8 @@ if test "x$CXXFLAGS_overridden" = "xno"; then
   AX_CHECK_COMPILE_FLAG([-Wunused-local-typedef],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-unused-local-typedef"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wdeprecated-register],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-deprecated-register"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wimplicit-fallthrough],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-implicit-fallthrough"],,[[$CXXFLAG_WERROR]])
+  dnl Temporary workaround for QTBUG-75210 on Qt 5.12 until it is fixed.
```

Comment:
This bug is also in qt 5.13.2 (not sure if it is tracked)

Reply:
The patch is [merged](https://codereview.qt-project.org/c/qt/qtbase/+/245434) in Qt 5.13 tree though.

Reply:
Is removing of mentioning of "Qt 5.12" will be ok?

Reply:
I was testing on Fedora 32, which comes with qt 5.13 and the issue is still there

Reply:
> I was testing on Fedora 32, which comes with qt 5.13 and the issue is still there

Confirm.